src/d3a/models/strategy/__init__.py

```diff
@@ -330,6 +330,33 @@ def post_offer(self, market, replace_existing=True, **offer_kwargs) -> Offer:
 
         return offer
 
+    def post_first_offer(self, market, energy_kWh, initial_energy_rate):
+        if not all(offer.seller != self.owner.name for offer in market.get_offers().values()):
+            self.owner.log.warning("There is already another offer posted on the market, therefore"
+                                   " do not repost another first offer.")
+            return None
```

Comment:
```suggestion
            return
```

Any specific advantage having an explicit None as return?

Reply:
Not really, apart from matching the post_first_bid method which more or less does the same. Will change both methods to only return.